Backend/.env

```
@@ -0,0 +1,2 @@
+MONGODB_URL=mongodb+srv://aniketkumar07c_db_user:iQaKdtvKCQT9xT2v@cluster0.aomotrx.mongodb.net/
+JWT_SECRET=your_jwt_secret_key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)